microcosm_eventsource/tests/fixtures.py

```diff
@@ -269,6 +329,57 @@ def event_factory(self):
         )
 
 
+@binding("immutable_task_event_controller")
+class ImmutableTaskEventController(EventController):
+    def __init__(self, graph):
+        super(ImmutableTaskEventController, self).__init__(
+            graph,
+            store=graph.immutable_task_event_store
+        )
+        self.ns = Namespace(
+            subject=ImmutableTaskEvent,
+            version="v1",
+        )
+        self.immutable_task_event_factory = graph.immutable_task_event_factory
+
+    def get_event_factory_kwargs(self):
+        return {}
+
+    @property
+    def event_factory(self):
+        return self.immutable_task_event_factory
+
+
+@binding("immutable_task_event_factory")
+class ImmutableTaskEventFactory(ImmutableEventFactory):
+    def __init__(self, graph):
+        super().__init__(
+            event_store=graph.immutable_task_event_store,
+            container_store=graph.immutable_task_store,
+            identifier_key="immutable_task_event_id",
+            default_ns=Namespace(
+                subject=ImmutableTaskEvent,
+                version="v1",
+            ),
+        )
+
+    @register_container_mutator_common(event_type=ImmutableTaskEvent)
+    def update_latest_task_event(self, container, event):
+        container.latest_task_event = str(event.event_type)
+
+    @register_container_mutator_by_event_type(event_instance_type=ImmutableTaskEventType.ASSIGNED)
```

Comment:
If you don't want to go with my suggestion of using one decorator for both event types and event_instance types, then maybe call this decorator `event_type_aggregator`?